### PR TITLE
Base impl converting Firestore data to protos

### PR DESCRIPTION
**Pull request by gino-m**
Towards #1758.

### CODE REVIEW

**Comment by gino-m** [comment]
@rfontanarosa @DaoyuT PTAL?

File: lib/src/proto-to-firestore.ts [hunk 1]
```diff
@@ -14,33 +14,35 @@
...
+  message: any,
+  descriptor: MessageDescriptor
+): DocumentData {
   const firestoreMap: DocumentData = {};
   for (const name in descriptor.fields) {
     const fieldDescriptor = descriptor.fields[name];
     const fieldNumber = fieldDescriptor?.id;
     if (!fieldNumber) {
-      console.debug(`Skipping unknown field $name in $type`);
+      console.debug('Skipping unknown field $name in $type');
```
Comment (rfontanarosa) [inline lib/src/proto-to-firestore.ts:45]: is this what you want to print? 'Skipping unknown field $name in $type'

**Review by rfontanarosa — CHANGES_REQUESTED** [review]

File: lib/src/proto-to-firestore.ts (hunk 1 above)
Comment (gino-m) [reply lib/src/proto-to-firestore.ts]: Sorry, I got a case of Kotlin-brain. Fixed.

**Comment by gino-m** [comment]
@rfontanarosa @DaoyuT RFAL!

**Pull request merged**